test/runnable/aliasassign.d

```diff
@@ -0,0 +1,31 @@
+
+
+template AliasSeq(T...) { alias AliasSeq = T; }
+
+template staticMap(alias F, T...)
+{
+    alias A = AliasSeq!();
+    static foreach (t; T)
```

Comment:
This is still gonna create `T.length - 1` number of intermediate instances of `AliasSeq`. Would be interesting to see how its compilation performance compares to a solution using alias functions, @UplinkCoder.

Reply:
> This is still gonna create T.length - 1 number of intermediate instances of AliasSeq.

Yup. A lot less than T.length-1 number of template instantiations of staticMap, which are far more expensive.